spanner/spanner_snippets/spanner/spanner_update_database.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package spanner
+
+// [START spanner_update_database]
+import (
+	"context"
+	"fmt"
+	"io"
+	"regexp"
+
+	database "cloud.google.com/go/spanner/admin/database/apiv1"
+	adminpb "cloud.google.com/go/spanner/admin/database/apiv1/databasepb"
+	"google.golang.org/genproto/protobuf/field_mask"
+)
+
+func updateDatabase(ctx context.Context, w io.Writer, db string) error {
+	matches := regexp.MustCompile("^(.*)/databases/(.*)$").FindStringSubmatch(db)
```

Comment:
I'm not sure this check belongs in the code sample. Typical practice is to put an example of what the variable should be in the function body, e.g.:
```
// db := "projectID/databases/db-name"
```
(or whatever the expected format would be) so that the user can uncomment and sub their own.

Reply:
@tritone 
Sure, I followed the existing samples that had this check https://github.com/GoogleCloudPlatform/golang-samples/blob/main/spanner/spanner_snippets/spanner/spanner_create_database.go#L29.
I agree with you that putting an example is better in samples. Have made relevant change.